testsuite/python/tests_common.py

```diff
@@ -638,6 +633,13 @@ def gay_berne_potential(r_ij, u_i, u_j, epsilon_0, sigma_0, mu, nu, k_1, k_2):
     return 4. * epsilon * (rr**-12 - rr**-6)
 
 
+class DynamicDict(dict):
```

Comment:
looks like a cherry-pick went wrong here, we definitively don't want `DynamicDict` back ;-)

Reply:
I'll look what went wrong with tests_common